etc/bankdefs/hipo4/neuralnetwork.json

```diff
@@ -1,4 +1,23 @@
 [ 
+	{
```

Comment:
Can the new bank be added at the bottom of the file, so that they are ordered by item?